php/admin/class-import.php

```diff
@@ -204,7 +204,7 @@ public function validate_upload( $file ) {
 	 *
 	 * @param array $blocks An array of Block Lab content blocks.
 	 */
-	public function import( $blocks ) {
+	public function import_blocks( $blocks ) {
```

Comment:
There was a warning that this method of `import()` has the same name as the class, `Import`, and could be confused with a constructor.

Reply:
This will require regression testing of importing, to make sure it doesn't break anythign.